bindings/go/evmc/evmc.go

```diff
@@ -149,6 +149,8 @@ const (
 	Istanbul         Revision = C.EVMC_ISTANBUL
 	Berlin           Revision = C.EVMC_BERLIN
 	London           Revision = C.EVMC_LONDON
+	Shanghai         Revision = C.EVMC_SHANGHAI
+	maxRevision      Revision = C.EVMC_MAX_REVISION
 )
 
```

Comment:
Unrelated, but can't comment on the line: can `func (err Error) Error() string` be replaced by the new string helpers?